src/vos/evtree.c

```diff
@@ -950,6 +950,26 @@ evt_desc_free(struct evt_context *tcx, struct evt_desc *desc, daos_size_t size)
 	return rc;
 }
 
+static int
+evt_desc_free_all(struct evt_context *tcx, struct evt_node_entry *ne)
```

Comment:
evt_node_entry_free perhaps?   The name was a bit confusing as I expected it to free more than one evt_desc

Reply:
OK, I will unapt the patch for that.